src/game_objects/player.cpp

```diff
@@ -290,6 +291,22 @@ void Player::c_take_damage() {
         count++;
     });
 
+    // Change to the hurt animation
+    stunned = true;
+    anim_player.set_anim("damage");
```

Comment:
Should put all this "anim_player.set_anim(), loop, speed, and play" into a single function. (lol, that's partially my fault though). Either like "anim_player.play("damage", 1.0f, false)" or aa function on the player like "player_hit_anim()"